array-manipulation-3/src/intersection.js

```diff
@@ -0,0 +1,12 @@
+/* exported intersection */
+function intersection(first, second) {
+  var Array = [];
```

Comment:
JavaScript already has a variable named `Array`, it is the array constructor, so you don't want to use that as a variable name. Generally speaking variables shouldn't start with a capital letter.